app/javascript/packs/routes.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import {Router, browserHistory, Route, IndexRoute } from 'react-router';
+import SpotsIndexContainer from './containers/SpotsIndexContainer';
+import SpotsShowContainer from './containers/SpotsShowContainer';
+
+class Routes extends Component {
+
+  constructor(props){
+    super(props);
+    this.state = {}
+  }
+
+  render() {
+    return (
+      <Router history={browserHistory}>
+        <Route path='/' component={SpotsIndexContainer} />
```

Comment:
I would consider supplying a layout as part of this routing configuration to combine common UI elements (navbars, etc).